innoweb/tests.py

```diff
@@ -0,0 +1,112 @@
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from selenium.webdriver.common.by import By
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.options import Options
```

Comment:
Lines 4 and 5 should be deleted as they are imports unused.

Reply:
OK, fixed 👌🏻